sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -409,9 +407,6 @@ private[joins] class UnsafeHashedRelation(
     val pageSizeBytes = Option(SparkEnv.get).map(_.memoryManager.pageSizeBytes)
       .getOrElse(new SparkConf().get(BUFFER_PAGESIZE).getOrElse(16L * 1024 * 1024))
 
-    // TODO(josh): We won't need this dummy memory manager after future refactorings; revisit
```

Comment:
friendly ping @JoshRosen , can we remove these 2 TOOD?

Reply:
Thank you for your explanation. So, do we have a more suitable way to re-describe this TODO? We can change to truly track it by creating a Jira and updating the TODO description.